framework/src/outputs/Console.C

```diff
@@ -366,7 +366,7 @@ Console::output(const ExecFlagType & type)
       _old_nonlinear_norm = std::numeric_limits<Real>::max();
 
     _console << std::setw(2) << _nonlinear_iter
-             << " Nonlinear |R| = " << outputNorm(_old_nonlinear_norm, _norm) << '\n';
+             << " Nonlinear |R| = " << outputNorm(_old_nonlinear_norm, _norm) << std::endl;
```

Comment:
I imagine there would be a huge impact to codes like Relap-7 while almost none to Rattlesnake. @YaqiWang, are you really losing output, or just don't like the buffering?